Cargo.lock

```diff
@@ -7462,7 +7462,9 @@ dependencies = [
 name = "solana-unified-scheduler-logic"
 version = "1.19.0"
 dependencies = [
+ "assert_matches",
```

Comment:
btw, this will soon be not needed as par the progress at: https://github.com/rust-lang/rust/issues/82775